apps/tailwind-components/pages/Form.story.vue

```diff
@@ -199,6 +181,17 @@ const numberOfFieldsWithErrors = computed(
           class="overflow-y-auto max-h-[calc(95vh-232px)] min-w-1000px"
           :class="sections.length > 0 ? 'col-span-3' : 'col-span-4'"
         >
+          <Button
```

Comment:
Should these be a component? It might be difficult to keep the styles consistent across uses.

Reply:
hmm, that is a good question. So you propose to have 'semantic' components next to the 'base' components to ensure consistent styling. So this would be a 'SectionNavigationLink' ?

Reply:
done